tests/keymanager.test.ts

```diff
@@ -1,9 +1,11 @@
-import { KeyManagerLevel, User } from 'tonomy-id-sdk';
+import { KeyManagerLevel, initialize } from 'tonomy-id-sdk';
 import { PrivateKey } from '@greymass/eosio';
 import JsKeyManager from './services/jskeymanager';
+import MockStorage from './services/mockstorage';
```

Comment:
shall we call it the JsStorage?

Reply:
it may be useful not just in mock testing

Reply:
done